usr/share/rear/conf/default.conf

```diff
@@ -3247,6 +3247,47 @@ VEEAM_BACKUPID="${VEEAM_BACKUPID:-}"
 # End of BACKUP=VEEAM default settings.
 ####
 
+# BACKUP=COVE (Cove Data Protection; Backup Manager)
+##
+# ReaR support for Cove Data Protection (Cove).
+# The external software component Cove is supported and maintained by N-able.
+
+REQUIRED_PROGS_COVE=(
+    base64
+    chgrp
+    dirname
+    ln
+    sleep
+    /bin/kill
+)
+
+# Let the Backup Manager know that it is installing or running inside the rescue environment.
+KERNEL_CMDLINE_COVE="cove_rescue_media"
```

Comment:
All `COVE` related variables must start from `COVE_` so that `rear dump` will show them properly